python/src/LinearModelAnalysis_doc.i.in

```diff
@@ -98,9 +122,21 @@ fisherPValue : float
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::LinearModelAnalysis::getResidualsStandardError
+"Accessor to the standard error of the residuals.
```

Comment:
should we write the formula of this ?
I'm asking because I had a similar problem with #2494

Reply:
Ok : fixed at 613a05303

Reply:
Ok. Fixed at c3e9ed2327abddbf10cd2299e41636b70caa6f12